managed/ui/src/components/config/PublicCloud/Kubernetes/CreateKubernetesConfiguration.js

```diff
@@ -270,7 +276,8 @@ class CreateKubernetesConfiguration extends Component {
                         <Col lg={7}>
                           <Field
                             name="imageRegistry"
-                            placeholder="quay.io/yugabyte/yugabyte"
+                            placeholder={ providerTypeOptions.length === 1 && providerTypeOptions[0].value === 'openshift'
+                                          ? "registry.connect.redhat.com/yugabytedb/yugabyte" : "quay.io/yugabyte/yugabyte" }
```

Comment:
let's put these urls into a `const` and then reuse it

Reply:
I have added them as two variables, what do you think?

Reply:
Or should I add it something like:
```js
const containerRegistries = { quay: 'quay.io/yugabytedb/yugabyte', redhat: 'registry.connect.redhat.com/yugabytedb/yugabyte' };
```